docs/source/checks/tabular/model_evaluation/plot_segment_performance.py

```diff
@@ -34,4 +34,4 @@
 
 from deepchecks.tabular.checks import SegmentPerformance
 
-SegmentPerformance(feature_1='workclass', feature_2='hours-per-week').run(validation_ds, model)
+SegmentPerformance(feature_1='workclass', feature_2='hours-per-week').run(validation_ds, model)
```

Comment:
?

Reply:
maybe i deleted it by accident and then pasted it.
it's the same i see. does it matter?

Reply:
The comment here is the fact that you removed the last blank line at the end of the file.
It matters just because some interpreters and software don't like files without a blank line at the end of the file. So you can just add it back, although it feels stupid sometimes 😅